sdk/core/core-websockets/test/utils/utils.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { TestContext } from "vitest";
+
+export function createIdentifier(test: TestContext): string {
```

Comment:
Don't we have something similar to this for recordings in the test-utils-vitest?  if not, shouldn't it be there?

Reply:
I couldn't find anything resembling this there so I moved it there and named it `createTestFullName`.